src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -10,6 +10,15 @@
     type: group
     short-summary: Manage Azure locks.
 """
+helps['lock update'] = """
```

Comment:
This should be in the YAML format like the rest of the entries in the file. I'm actually kind of surprised it doesn't throw an error.